### PR TITLE
Private count tests added for models

### PR DESCRIPTION
**Pull request by sambuddhabasu**
Models tests have been updated to take into account the private flag with private and non-private count tests.

### CODE REVIEW

File: bookie/tests/test_models/test_bmarkmgr.py [hunk 1]
```diff
@@ -123,6 +123,44 @@ def test_per_user(self):
         ct = BmarkMgr.count(username=usercommon.username)
         self.assertEqual(2, ct, 'We should have a total of 2: ' + str(ct))
 
+    def test_per_user_is_private(self):
```
Comment (mitechie) [inline bookie/tests/test_models/test_bmarkmgr.py]: test_per_user_accounts_for_privacy

File: bookie/tests/test_models/test_bmarkmgr.py [hunk 2]
```diff
@@ -123,6 +123,44 @@ def test_per_user(self):
...
+        d = Bmark(
+            url=gen_random_word(10),
+            username=usercommon.username,
+            is_private=False,
+        )
+        DBSession.add(d)
+        DBSession.flush()
+
+        ct = BmarkMgr.count(username=usercommon.username, is_private=False)
+        self.assertEqual(2, ct, 'We should have a total of 2: ' + str(ct))
```
Comment (mitechie) [inline bookie/tests/test_models/test_bmarkmgr.py]: "We should only have 2 private bookmarks"

Comment (sambuddhabasu) [inline bookie/tests/test_models/test_bmarkmgr.py]: Shouldn't the string be "We should only have 2 public bookmarks" ?

Comment (mitechie) [inline bookie/tests/test_models/test_bmarkmgr.py]: oh yea, it'd help if I read things properly

**Comment by preddy5** [comment]
:+1:

**Comment by mitechie** [comment]
This branch looks good. We should file a bug about the pgsql failures to investigate. It seems it started to get flaky after the migration to add is_private. Can you file that bug and add a card for that bug to trello we can look at soon?

Thanks for the change.

**Comment by sambuddhabasu** [comment]
So, do you think that this branch is good to be merged?
Also, I will file a bug shortly and also update the trello board.

**Comment by mitechie** [comment]
Sorry yes to land. :+1:

**Comment by sambuddhabasu** [comment]
:shipit:

**Pull request merged**